docs/book/src/cronjob-tutorial/testdata/project/api/v1/zz_generated.deepcopy.go

```diff
@@ -52,6 +52,36 @@ func (in *CronJob) DeepCopyObject() runtime.Object {
 	return nil
 }
 
+// DeepCopyInto is an autogenerated deepcopy function, copying the receiver, writing into out. in must be non-nil.
```

Comment:
There is no need to generate deepcopy funcs for these types (I would also imagine that to be very hard / impossible in some cases (e.g. if folks add fields like a Client)

Reply:
That is generated automatically via controller-gen.
We have no control about that.

Reply:
This should be possible with these markers: "+kubebuilder:object:generate": https://book.kubebuilder.io/reference/markers/object.html?highlight=kubebuilder%3Aobject%3Agenerate#objectdeepcopy

Reply:
I forgot of this !!!
Good point.